msi/GitProduct.wxs

```diff
@@ -295,9 +315,12 @@
       <Custom Action="ExecConfigureCoreAutoCrlf" Before="ExecConfigureFsCache">
         <![CDATA[REMOVE <> "ALL"]]>
       </Custom>
-      <Custom Action="ExecConfigureFsCache" Before="ExecPostInstallBat">
+      <Custom Action="ExecConfigureFsCache" Before="ExecConfigureGcm">
         <![CDATA[REMOVE <> "ALL" AND ENABLEFSCACHE = "true"]]>
       </Custom>
+      <Custom Action="ExecConfigureGcm" Before="ExecPostInstallBat">
+        <![CDATA[REMOVE <> "ALL" AND WIX_IS_NETFRAMEWORK_451_OR_LATER_INSTALLED AND ENABLEGCM]]>
```

Comment:
We don't allow modifying the installation of Git for Windows via the MSI. We set `ARPNOMODIFY=1`. If a user wants to later "disable" Git Credentials Manager for Windows, they would need to run `git config -f $ProgramData/Git/config --unset credential.helper`.